clang/docs/ReleaseNotes.rst

```diff
@@ -385,6 +385,8 @@ related warnings within the method body.
 - Clang now disallows the use of attributes applied before an
   ``extern template`` declaration (#GH79893).
 
+- Clang now diagnoses unknown attribute namespaces
```

Comment:
```suggestion
- Clang now diagnoses unknown attribute namespaces.
```

Reply:
@erichkeane I’ve updated PR to use existing diagnostic with scope